sql/src/planner.rs

```diff
@@ -735,6 +745,15 @@ impl<'a, P: MetaProvider> PlannerDelegate<'a, P> {
             .context(MetaProviderFindTable)?
             .with_context(|| TableNotFound { name: table_name })
     }
+
+    fn find_option_table(&self, table_name: TableName) -> Result<Option<TableRef>> {
```

Comment:
Maybe naming it `find_table` is better?

Reply:
Should we pass &TableName or &str?
Clone here is strange.

Reply:
`find_table` already exists.

Reply:
I remove this function, and modify `find_table`.